fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -627,6 +619,9 @@ public static MaterializedView read(DataInput in) throws IOException {
      * @return
      */
     public boolean isLoadTriggeredRefresh() {
+        if (this.refreshScheme.getType() == MaterializedView.RefreshType.INCREMENTAL) {
```

Comment:
what ignore refresh for `INCREMENTAL` MV? The data loading should trigger the incremental maintenance

Reply:
this is not ignore for INCREMENTAL.  This will tigger for `INCREMENTAL ` MV for each refresh.